Task9.1.js

```diff
@@ -11,24 +13,48 @@ function lastVariant () {
 function cutExtraSymbols (stroka) {
   let newStr = Array.from(stroka);
   for (let x=0; x <= (stroka.length-1); x++){
-      let massive1 = splitStrOnWords(stroka);
+      let massive1 = indexInCase (stroka);
       if ( massive1.includes( stroka[x].toLowerCase()) ) {
           newStr.splice(x, 1, '');
       }
   } return  newStr.join('');
 };
 
-function splitStrOnWords(line) { // строка разбивается на слова и по каждому слову ищутся неуникальные символы
-    let ready = [];
-    let strWords = line.split(' '); 
-    for (let j=0; j <= (strWords.length-1); j++) {
-      let strSymbols = strWords[j].split(''); 
+// function splitStrOnWords(line) { // строка разбивается на слова и по каждому слову ищутся неуникальные символы
```

Comment:
Plus, worth saying, there's a good practice not to leave commented code chunks (dead code) in the repo.